src/ConfigWidgets/SymbolsDialogTest.cpp

```diff
@@ -39,12 +39,19 @@ class MockPersistentStorageManager : public orbit_symbol_paths::PersistentStorag
               LoadModuleSymbolFileMappings, (), (override));
 };
 
+void SetUpExpectMockCalls(MockPersistentStorageManager* mock_storage_manager,
+                          std::vector<std::filesystem::path> load_paths = {},
+                          absl::Span<const std::filesystem::path> save_paths = {}) {
+  EXPECT_CALL(*mock_storage_manager, LoadPaths).WillOnce(testing::Return(std::move(load_paths)));
+  EXPECT_CALL(*mock_storage_manager, SavePaths)
+      .WillOnce([save_paths](absl::Span<const std::filesystem::path> paths) {
```

Comment:
Here you store a copy of the span. That's not great because the span could be created from a temporary, so this is a potential dangling pointer.

Reply:
used std::vector

Reply:
used std::vector instead